.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -7,19 +7,26 @@ about: Request a new feature
 <!--
 DELETE ME
 
-Use the search function to see if someone else has already submitted the same feature request. If there already is another request, please upvote the first post instead of commenting something like "I also want this".
+Make sure you are using the latest version of AntennaPod. Perhaps the feature you are looking for has already been implemented.
+
+Use the search function to see if someone else has already submitted the same feature request. If there is another request already, please upvote the first post instead of commenting something like "I also want this".
 
 To make it easier for us to keep track of requests, please only make one feature request per issue.
 
-Give a brief explanation about the problem that currently exists and why your requested feature solves this problem.
+Give a brief explanation about the problem that may currently exist and how your requested feature solves this problem.
 
-Try do be as specific as possible. Please not only explain what the feature does, but also how. When your request is about changing or extending the UI, describe what the UI would look like and how the user would interact with it.
+Try to be as specific as possible. Please not only explain what the feature does, but also how. If your request is about (or includes) changing or extending the UI, describe what the UI would look like and how the user would interact with it.
 -->
 
-**App version**: 1.x (from Google Play/F-Droid/Custom build)
+<!-- READ THIS: The latest version may be different depending on your device. You can find the version in AntennaPod's settings. -->
+<!-- Because of the above, PLEASE DO NOT write "latest" as the app version, as it is not helpful! -->
```

Comment:
This sounds pretty aggressive. I would remove this line and just keep the one above.